app/app.go

```diff
@@ -698,6 +702,15 @@ func (app *App) setPostHanders() {
 	app.SetPostHandler(postHandler)
 }
 
+func (*App) ForbiddenParams() [][2]string {
```

Comment:
```suggestion
func (*App) ForbiddenParams() [][2]string {
```
nit but forbidden sounds like there's possibly something dangerous about these parameters whereas this component simply locks their value (they cannot be changed). Would suggest calling them `LockedParams()`

Reply:
More of a note here but I guess we could version this by app version so we are able to modify the list over time

Reply:
I like the rationale for `LockedParams()` instead of `ForbiddenParams()` but the new name makes me think the param can be unlocked and modified via governance. Alternative naming ideas: `ImmutableParams()` or `HardCodedParams()`

Reply:
good points, switched to `BlockedParams` in https://github.com/celestiaorg/celestia-app/pull/1694/commits/1a9506617fd5d49b6e7d26a5d8b990c420f0a72d